react/game/src/app/screens/Login/index.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+
+import LoginLayout from './layout';
+
+class LoginContainer extends Component {
+  handleSubmit = data => {
+    console.log(data);
```

Comment:
A good approach is put a `//TODO: some text` to indicate and inform in the pull request that this will change